stock_landed_cost_company_percentage/models/res_company.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    journal_id = fields.Many2one(
```

Comment:
rename the field, this is too generic and may conflict with other addons.

Reply:
Corresponded. Thank you.